packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -539,8 +546,23 @@ export class SummarizerNode implements IRootSummarizerNode {
         if (this.isTrackingInProgress()) {
             child.wipReferenceSequenceNumber = this.wipReferenceSequenceNumber;
         }
+        // In case we have pending summaries on the parent, let's initialize it on the child.
+        if (child._latestSummary !== undefined) {
```

Comment:
Now that I think about it, this block should be moved up into the if statement this.isTrackingInProgress()